depends.txt

```diff
@@ -1 +1 @@
-default
+default
```

Comment:
Linux, Win and Mac have different conventions of line ending. Namely, Win uses CR LF ("\r\n" in C), while Mac uses CR only, and Linux uses LF only. But here, that’s not the problem: I checked in my viewer; your version of `depends.txt` just lacks line ending character at all.